src/components/Spell.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import "../css/Spell.css";
+
+const Spell = ({ name }) => {
+  return (
+    <div className="spell">
+      <h4>{`💥 ${name}`}</h4>
+    </div>
+  )
+}
```

Comment:
I think the Spell component also needs a key?  

Also, I think `<div className="spell"></div>` (with maybe a small amount of editing for css purposes) is probably a better place to wrap the `<Link></Link>`.  I'm probably wrong though.  I'm famous for being wrong.

Reply:
You might be right! I just modeled this after what we did for tomatillos, which was wrapping the link when doing the render with map. But I think it can go either place probably? 
There is a key for Spell, it is also in the map in SpellBox.